website/docs/reference/warehouse-profiles/snowflake-profile.md

```diff
@@ -106,7 +106,7 @@ The "base" configs for Snowflake targets are shown below. Note that you should a
 
 | Config | Required? | Description |
 | ------ | --------- | ----------- |
-| account | Yes | The account to connect to. This will be something like `abc123` or `abc123.us-east-1` for your particular account |
+| account | Yes | The account to connect to. For AWS accounts in the default US West region, this will be something like `abc123` (without any other segments). For AWS accounts *not* in the default US West region and for GCP and Azure-based accounts, you also have to append the cloud platform, such as `aws`, `gcp` or `azure`, respectively (for example: `abc123.us-central1.gcp`) See [https://docs.snowflake.com/en/user-guide/intro-regions.html#specifying-region-information-in-your-account-hostname](https://docs.snowflake.com/en/user-guide/intro-regions.html#specifying-region-information-in-your-account-hostname) for more information. |
```

Comment:
This is correct; thanks so much for linking to the Snowflake docs!

Reply:
Alternatively, we could add a troubleshooting section as I did with [the Gitlab docs?](https://docs.getdbt.com/docs/dbt-cloud/cloud-configuring-dbt-cloud/connecting-gitlab#troubleshooting)

Reply:
How about we defer refactoring the whole section to another PR? I don't think I will quite have the time to think the whole approach here, so this may be out of scope for me.

Reply:
I can take this on!